### ISSUE TITLE
validator for reproin

### ISSUE THREAD

**Issue by satra** [ReproNim/reproin#26]
at present it's hard for people to debug via heudiconv whether the reproin naming scheme they adopted on the scanner is valid. would it be possible to add a validator to heudiconv that can simply go through the dicom grouping and check if the names are reproin compliant. or have heudiconv give better error messages when reproin heuristic is used.

**Comment by yarikoptic** [comment]
Yeah, I wanted something like that too. Eventually we could compliment it with ocr of the photo from console display ;-)

**Comment by yarikoptic** [comment]
note: not a validator but rather "constructor" was created - https://github.com/NPACore/reproin-namer
